logs/lib/retrieveLogs.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
   getLogs() {
     const project = this.serverless.service.provider.project;
     let func = this.options.function;
-    const pageSize = this.options.count || 10;
+    const pageSize = parseInt(this.options.count, 10) || 10;
```

Comment:
I think it worked previously correctly without casting to int, but I think it doesn't hurt here :+1: